core-web/libs/portlets/edit-ema/portlet/src/lib/dot-ema-shell/dot-ema-shell.component.ts

```diff
@@ -161,33 +159,27 @@ export class DotEmaShellComponent implements OnInit, OnDestroy {
         this.#currentComponent = event;
     }
 
-    handleNgEvent({ event }: { event: CustomEvent }) {
+    handleNgEvent({ event, dirty, saved, isTranslation }: DialogAction) {
         switch (event.detail.name) {
             case NG_CUSTOM_EVENTS.DIALOG_CLOSED: {
-                if (!this.$didTranslate()) {
+                // We dont want to reload if it was saved without changes and is not a translation
+
+                if ((dirty && saved) || (isTranslation && saved)) {
```

Comment:
When you have conditions like this, you need to ask yourself if your state management is correct.

Reply:
I'm changing this to an `status` as Arcadio recommended